daisy_workflows/image_import/debian/translate.py

```diff
@@ -48,6 +48,10 @@ def DistroSpecific(g):
 
   if install_gce == 'true':
     logging.info('Installing GCE packages.')
+
+    g.command(['apt-get', 'update'])
+    g.command(['apt-get', 'install', '-y', 'gnupg'])
```

Comment:
g.sh('DEBIAN_FRONTEND=noninteractive apt-get install --assume-yes gnupg)
This is to avoid any user input requests that may come up with different environments.

Reply:
Is g.command(['apt-get', 'update']) still needed?

Reply:
Yes, since this is happening before the next update (and repo additions).

Reply:
Done